src/components/tab/Tab.tsx

```diff
@@ -3,12 +3,22 @@ import { Tab as ChakraTab, TabProps } from '@chakra-ui/react';
 
 import { Space } from 'src/themes';
 
+import Link, { LinkProps } from '../link';
+
 type Props = {
   marginX?: Space;
+  isLink?: boolean;
+  linkProps?: LinkProps;
```

Comment:
I think you should be able to just add `href` here and then use it like
`<Tab as={NextLink} href="foo.ch" />` in the project. At least it worked for me in an experiment recently: https://github.com/smg-automotive/components-pkg/pull/898/files

Reply:
Yea, that could work too. I thought maybe to omit that as a whole. Remove that part from tab component. And implement it on the listing-web side. We would just wrap `Tab` component in `NextLink` or `Link`.
`
<NextLink>
    <Tab>...</Tab>
</NextLink>
`
What do you think about that?

Reply:
even better

Reply:
Works!